Decred Wallet/Features/Send/SendViewController.swift

```diff
@@ -503,33 +503,37 @@ extension SendViewController {
     }
     
     func checkAddressFromQrCode(textScannedFromQRCode: String?) {
-        guard var capturedText = textScannedFromQRCode else {
+        
+        guard let capturedText = textScannedFromQRCode else {
             self.destinationAddressTextField.text = ""
             return
         }
+        let URI: DecredAddressURI = DecredAddressURI(uriString: capturedText)
         
-        if capturedText.starts(with: "decred:") {
-            capturedText = capturedText.replacingOccurrences(of: "decred:", with: "")
+        guard let address = URI.address else {
+            return
```

Comment:
Previously, this function accepts a plain address string even if it does not begin with `decred:`.

Reply:
resolved in `10b8349`